collectors/aws/collector.js

```diff
@@ -230,6 +230,13 @@ var calls = {
                 ]
             }
         },
+        describeVpcEndpointServices: {
+            property: 'ServiceDetails',
+            paginate: 'NextToken',
+            filters: {
+                serviceName: 'ec2'
+            }
```

Comment:
These filters will need to be passed as params:
```suggestion
            params: {
                Filters: [
                    Name: 'serviceName',
                    Values: [
                        'ec2'
                    ]
                ]
            }
```

Reply:
But I actually don't think you need this filter. The filter was for the name of the VPC endpoint service, not the name of the AWS service (i.e. `ec2`). You should be able to remove this param entirely.